adminSiteClient/GdocsStore.tsx

```diff
@@ -23,36 +27,42 @@ export class GdocsStore {
         await this.admin.requestJSON(`/api/gdocs/${id}`, {}, "PUT")
     }
 
-    async update(gdoc: OwidArticleType) {
-        await this.admin.requestJSON(`/api/gdocs/${gdoc.id}`, gdoc, "PUT")
+    async update(gdoc: OwidArticleType): Promise<OwidArticleType> {
+        return this.admin
+            .requestJSON<OwidArticleTypeJSON>(
+                `/api/gdocs/${gdoc.googleId}`,
+                gdoc,
+                "PUT"
+            )
+            .then(getArticleFromJSON)
     }
 
-    async publish(gdoc: OwidArticleType) {
-        const publishedGdoc = {
+    async publish(gdoc: OwidArticleType): Promise<OwidArticleType> {
+        const publishedGdoc = await this.update({
```

Comment:
Using the response object as it has the correct `updatedAt` value from SQL's `CURRENT_TIMESTAMP`